pages/index.js

```diff
@@ -13,127 +12,126 @@ const Index = () => (
-            Open in a new tab
-          </a>
-        </li>
-        <li>
-          There is an exception during React lifecycle that is caught by
-          Next.js's React Error Boundary. In this case, when the component
-          mounts. This should cause _error.js to render and record Error('Client
-          Test 6') in Honeybadger.{' '}
-          <Link href="/client/test6">
-            <a>Perform client side navigation</a>
-          </Link>{' '}
-          or{' '}
-          <a href="/client/test6" target="_blank">
-            Open in a new tab
-          </a>
-        </li>
-        <li>
-          There is an unhandled Promise rejection during React lifecycle. In
-          this case, when the component mounts. Honeybadger should record
-          Error('Client Test 7').{' '}
-          <Link href="/client/test7">
-            <a>Perform client side navigation</a>
-          </Link>{' '}
-          or{' '}
-          <a href="/client/test7" target="_blank">
-            Open in a new tab
-          </a>
-        </li>
-        <li>
-          An Error is thrown from an event handler. Honeybadger should record
-          Error('Client Test 8').{' '}
-          <Link href="/client/test8">
-            <a>Perform client side navigation</a>
-          </Link>{' '}
-          or{' '}
-          <a href="/client/test8" target="_blank">
-            Open in a new tab
-          </a>
-        </li>
-      </ul>
-    </ul>
+    <h3>Server exceptions</h3>
+    <ol>
+      <li>
+        getInitialProps throws an Error. This should cause _error.js to render
+        and record Error('Server Test 1') in Honeybadger.{' '}
+        <a href="/server/test1" target="_blank">
+          Open in a new tab
+        </a>
+      </li>
+      <li>
+        getInitialProps returns a Promise that rejects. This should cause
+        _error.js to render and record Error('Server Test 2') in Honeybadger.{' '}
+        <a href="/server/test2" target="_blank">
+          Open in a new tab
+        </a>
+      </li>
+      <li>
+        getInitialProps calls a Promise that rejects, but does not handle the
+        rejection or await its result (returning synchronously).{' '}
+        Honeybadger shouldrecord Error('Server Test 3').{' '}
+        <a href="/server/test3" target="_blank">
+          Open in a new tab
+        </a>
+      </li>
+      <li>
+        There is a top-of-module Promise that rejects, but its result is not
+        awaited. Honeybadger should record Error('Server Test 4'). Note this will
+        also be recorded on the client side, once the page is hydrated.{' '}
+        <a href="/server/test4" target="_blank">
+          Open in a new tab
+        </a>
+      </li>
+    </ol>
+
+    <h3>Client exceptions</h3>
+    <ol>
+      <li>
+        getInitialProps throws an Error. This should cause _error.js to render
+        and record Error('Client Test 1') in Honeybadger.{' '}
+        <Link legacyBehavior href="/client/test1">
```

Comment:
Adding legacyBehavior should make these links work in NextJS 13 as well as older versions.